projects/plugins/boost/app/assets/src/js/pages/getting-started/GettingStarted.svelte

```diff
@@ -12,19 +15,84 @@
 
 	// svelte-ignore unused-export-let - Ignored values supplied by svelte-navigator.
 	export let navigate, location;
+
+	const initiatingFreePlan = writable( false );
+	const initiatingPaidPlan = writable( false );
+	const dismissedSnackbar = writable( false );
+
+	const snackbarMessage = derived(
+		[ connection, dismissedSnackbar ],
+		( [ $connection, $dismissedSnackbar ] ) => {
+			if ( ! ( $dismissedSnackbar || $connection.connected ) && $connection.error?.message ) {
+				if ( $connection.error instanceof ApiError ) {
+					return $connection.error.getDisplayBody();
+				}
+
+				return $connection.error.message;
+			}
+
+			return null;
+		}
+	);
```

Comment:
This seems like way too much ApiError-specific logic for the header of an unrelated svelte file. Also, I don't think you want `getDisplayBody` here - I think you just want `.message`. DisplayBody JSON-stringifies it.

Reply:
I have removed the usage of `getDisplayBody`. However, I didn't fully get what you mean by `snackbarMessage` being in an unrelated file and what would be the alternative. But, I have also rearranged where the Snackbar component is used.